settings.gradle

```diff
@@ -71,6 +71,7 @@ include 'internal:alpini:router:alpini-router-impl'
 // 3rd-party system integration modules
 include 'integrations:venice-pulsar'
 include 'integrations:venice-samza'
+include 'integrations:venice-beam'
```

Comment:
Thanks for adding the new module here. Just a minor nitpick: can we put it at the top of the list, so it's sorted alphabetically?

Also, if you would like, you could likewise add it to the `navigating_project.md` file along with a one-line description (e.g. "`venice-beam`, which implements the Beam `Read` API, enabling a Beam job to consume the Venice changelog.", or something to that effect...). This will make it show up in the docs at venicedb.org :D ...